re_run_gridjob.sh

```diff
@@ -8,55 +8,81 @@ export X509_USER_PROXY=/var/tmp/${USER}.${ROLE}.proxy
 source /e906/app/software/script/setup-jobsub-spinquest.sh
 
 dir_scripts=$(dirname $(readlink -f $BASH_SOURCE))
-dir_recofile=/pnfs/e1039/persistent/cosmic_recodata/$jobname
+#dir_recofile=/pnfs/e1039/persistent/users/apun/test_reco
+dir_recofile=/pnfs/e1039/persistent/cosmic_recodata
 
 #loop over submitted runs after ~1 hour of job submission
-while read -r RunNum N_splits reco_status; do
-    
-    #[[ $RunNum = \#* ]] && paste <(echo RunNum) <(echo N_splits) <(echo reco_status)>>reco_status_tmp.txt && continue
+while read -r RunNum N_splits reco_status; do    
 
-    if [[ $reco_status -eq 1 ]]; then #&& continue
+     [[ $reco_status -ne 1 ]] && continue
 	
 	run_dir=($(printf 'run_%06d' $RunNum) )
 
 	N_GOOD_LOG=0
-	
-	
+		
 	#loop over the log files of the submitted runs
-	for i in $dir_recofile/$run_dir/*/log/log.txt; do
+	for i in $dir_recofile/$run_dir/$run_dir*; do
 	    echo $i
-	    [[ ! -e $i ]] && continue
+
+	    resub_file_dir=$(basename $i)
+	    echo $resub_file_dir
             
-	    job_status=$(tail -1 "$i" | head -1) #reco_status from root -l {macro} command in gridrun_data.sh	
+	    resub_file=$resub_file_dir'.root'
```

Comment:
This line looks strange since a directory name + '.root' becomes a file name.  A proper variable name is wanted.

Reply:
The file name is given as input while resubmitting the job as;
$dir_scripts/gridsub_data.sh $run_dir 1 $RunNum 0 splitting $resub_file

Reply:
My point was that `$resub_file_dir` is actually not a directory name but a base name (with path).  If so, a proper name should be given to this variable.  The present name is confusing and I had thought that this line was a bug.

Reply:
I see. I was wondering if I should change the name as following
$resub_file_dir => $file_base_dir
$resub_file => $file_name
Or, please suggest me and I will change accordingly.

Reply:
I would suggest `resub_file_base` or `resub_file_base_name` for `resub_file_dir`, although "base name" is not quite perfect since it usually doesn't include path.

`resub_file` seems fine as is.

Reply:
Thank you. I have changed it accordingly.